pkg/controller/istiocontrolplane/listeners.go

```diff
@@ -327,49 +325,3 @@ func (c *CitadelChartCustomizer) cleanCitadelResources(obj runtime.Object) error
 	}
 	return nil
 }
-
-// KialiChartCustomizer is a ChartCustomizer for the kiali chart
```

Comment:
we probably no longer want to customize KialiChart- but KialiOperator may desire customization...